.gitignore

```diff
@@ -24,5 +24,7 @@ _testmain.go
 *.prof
 .env
 
-coverage.out
+*.swp
+coverage*.out
```

Comment:
Just drop that change, the old ignore will be matched if you do it like the old makefile.